<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="cpuTempGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>120</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>GPU temp [Â°C]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="cpuUseGroupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>120</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>CPU usage [%]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="fanGroupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>120</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>GPU mem [10*GB]</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ramGroupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>120</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>RAM usage [%]</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="cpuTempVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="cpuUseVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="fanVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="ramVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="tempButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: 100%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="useButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: 100%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fanButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: 100%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ramButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold: 100%</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="prodLineGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>299</y>
      <width>581</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Production line</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>300</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>235</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Fast</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Slow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionSearch_path">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search path</string>
   </property>
  </action>
  <action name="actionLoad_last_path">
   <property name="text">
    <string>Load last path</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
